src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -278,14 +278,17 @@ def k8s_install_cli(client_version='latest', install_location=None):
 
     file_url = ''
     system = platform.system()
-    base_url = 'https://storage.googleapis.com/kubernetes-release/release/{}/bin/{}/amd64/{}'
+    base_url = 'https://storage.googleapis.com/kubernetes-release/release/v{}/bin/{}/{}/{}'
+    arch = platform.machine().lower()
```

Comment:
One my mac, `platform.machine()` returns `x86_64`.
`https://storage.googleapis.com/kubernetes-release/release/v1.6.3/bin/darwin/x86_64/kubectl` does not exist.